src/__snapshots__/index.test.ts.snap

```diff
@@ -15560,15 +15560,15 @@ Object {
       ],
       "tool": Object {
         "driver": Object {
-          "downloadUri": "https://www.npmjs.com/package/axe-core/v/3.2.2",
-          "fullName": "axe for Web v3.2.2",
+          "downloadUri": "https://www.npmjs.com/package/axe-core/v/undefined",
+          "fullName": "axe for Web vundefined",
```

Comment:
do you think this should be `vundefined` in the snapshot?

Reply:
It's covered by other tests, but yeah, it looks a little off. Will improve it.